kernel/memory/src/paging/table.rs

```diff
@@ -25,7 +25,7 @@ use zerocopy::FromBytes;
 /// See these links for more: 
 /// * <http://forum.osdev.org/viewtopic.php?f=1&p=176913>
 /// * <http://forum.osdev.org/viewtopic.php?f=15&t=25545>
-pub const P4: *mut Table<Level4> = 0o177777_776_776_776_776_0000 as *mut _; 
+pub const P4: *mut Table<Level4> = 0o1_777_777_767_767_767_760_000 as *mut _; 
```

Comment:
The spacing actually had a point, but I know clippy screams at you about this and it doesn't really matter much.